spec/features/admin/invoices/show_spec.rb

```diff
@@ -33,4 +32,42 @@
     expect(page).to have_content(invoice_item_1.quantity)
     expect(page).to have_content(invoice_item_3.status)
   end
+
+  scenario 'visitor sees invoice status is a select field next to invoice with current status selected' do
+    expect(page).to have_field('Status', with: 'completed')
+    expect(page).to have_button('Update Invoice Status')
+  end
+
+  scenario 'visitor can select new status and update by clicking submit button' do
+    expect(page).to have_field('Status', with: 'completed')
+    expect(page).to have_button('Update Invoice Status')
+
+    select 'Cancelled', from: 'Status'
+    click_button('Update Invoice Status')
+
+    expect(current_path).to eq(admin_invoice_path(invoice_1.id))
+    expect(page).to have_field('Status', with: 'cancelled')
+
+    select 'In Progress', from: 'Status'
+    click_button('Update Invoice Status')
+
+    expect(current_path).to eq(admin_invoice_path(invoice_1.id))
+    expect(page).to have_field('Status', with: 'in progress')
+
+    select 'Completed', from: 'Status'
+    click_button('Update Invoice Status')
+
+    expect(current_path).to eq(admin_invoice_path(invoice_1.id))
+    expect(page).to have_field('Status', with: 'completed')
+  end
+#   As an admin
+# When I visit an admin invoice show page
+# I see the invoice status is a select field
+# And I see that the invoice's current status is selected
+# When I click this select field,
+# Then I can select a new status for the Invoice,
+# And next to the select field I see a button to "Update Invoice Status"
+# When I click this button
+# I am taken back to the admin invoice show page
+# And I see that my Invoice's status has now been updated
```

Comment:
Just need to get rid of the user story, but looks fantastic!

Reply:
Agreed, looks great overall!

Reply:
Done in commit appended below.